.github/workflows/main.yml

```diff
@@ -10,8 +10,8 @@ jobs:
       fail-fast: false
       matrix:
         node-version:
+          - 20
           - 18
-          - 16
```

Comment:
16 should be preserved

Reply:
I have a question, [you made node 18 the minimum required version](https://github.com/sindresorhus/file-type/commit/7f4b30b2ef89f73279fb4ff95e5c38e9a0687181) for `file-type` which this library is a thin wrapper for. Would a followup be welcome to update to the latest major version of `file-type` and drop node 16 support?

Reply:
Yes